src/explore-education-statistics-admin/src/services/releaseDataFileService.ts

```diff
@@ -67,21 +67,15 @@ const releaseDataFileService = {
   uploadDataFiles(
     releaseId: string,
     request: UploadDataFilesRequest,
-  ): Promise<DataFile> {
+  ): Promise<void> {
```

Comment:
This isn't particularly good RESTful API design. Just because the frontend currently doesn't do anything with the result doesn't mean that the API shouldn't return anything.

Reply:
Yep good point - Slightly special case the datafiles upload so I have returned a boolean here.